addons/realisticnames/stringtable.xml

```diff
@@ -3344,12 +3344,14 @@
         </Key>
         <Key ID="STR_ACE_RealisticNames_optic_lrps">
             <English>Nightforce NXS</English>
+            <German>Nightforce NXS </German>
             <Chinese>Nightforce NXS</Chinese>
             <Chinesesimp>Nightforce NXS</Chinesesimp>
             <Japanese>Nightforce NXS</Japanese>
         </Key>
         <Key ID="STR_ACE_RealisticNames_optic_lrps_ghex">
             <English>Nightforce NXS (Green Hex)</English>
+            <German>Nightforce NXS (Grün Hex.) </German>
```

Comment:
Leerzeichen zu viel

Reply:
And the period before the closing parenthesis.

Reply:
`Grünes Hex`